code/unclassified/fifteen/fifteen.c

```diff
@@ -0,0 +1,291 @@
+/**
+ * fifteen.c
+ *
+ * Implements Game of Fifteen (generalized to d x d).
+ *
+ * Usage: fifteen d
+ *
+ * whereby the board's dimensions are to be d x d,
+ * where d must be in [DIM_MIN,DIM_MAX]
+ *
+ * Note that usleep is obsolete, but it offers more granularity than
+ * sleep and is simpler to use than nanosleep; `man usleep` for more.
+ 
+ * Author:- Rishav Pandey
+ */
+ 
+ 
+#define _XOPEN_SOURCE 500
+
+#include <cs50.h>
```

Comment:
Where is `cs50.h`? Additionally, we accept only library which are readily available and are not locally developed. Kindly modify your code to remove this dependency 👍

Reply:
You will find the instruction note for including cs50.h library. This is very simple yet efficient way to use the functions instead of scanf(). As we know scanf() has very adverse effect while using it in complex program.